### PR TITLE
Migrate Admin SCM Docs to Dev Center

### PR DESCRIPTION
**Pull request by ryanmillerdev**
Resolves [DS-518](https://algorithmia.atlassian.net/browse/DS-518). After discussions with @hernan-alvarez and @StephanieKim, we've decided the admin docs for SCM configuration should land in the devcenter. This PR talks all content from the prior Google Doc and adds it as a standalone page.

@StephanieKim, would like your thoughts on the change I made to the navmenu. It hides the section on enterprise docs for our public marketplace users, but the pages themselves are still viewable if users know the direct link.

### CODE REVIEW

**Review by stevielkim — APPROVED** [review]

**Comment by ryanmillerdev** [comment]
@bnlb Added you as a reviewer here as well–if you'd double-check my "enterprise navbar" edit, I'd be appreciative!

**Comment by bnlb** [comment]
> @bnlb Added you as a reviewer here as well–if you'd double-check my "enterprise navbar" edit, I'd be appreciative!

Started looking at this before our string of meetings yesterday and forgot to come back to it. Looking now!

File: _pages/algorithmia-enterprise/scms.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+<img src="{{site.cdnurl}}{{site.baseurl}}/images/post_images/admin_scms/disable_scm.png" alt="Set an SCM as default" class="screenshot img-sm">
+
+The SCM will now be available for use by all users within your Algorithmia instance.
+
+#### Selecting a Default SCM
+
+If you have multiple enabled SCMs, you may wish to guide your users to a specific SCM, such as Github. You can do this by selecting an enabled SCM and clicking the “Make Default” button. Once made default, your selected SCM will be the first SCM selected whenever a user attempts to create an algorithm.
+
+<img src="{{site.cdnurl}}{{site.baseurl}}/images/post_images/admin_scms/make_scm_default.png" alt="Set an SCM as default" class="screenshot img-sm">
```
Comment (bnlb) [inline _pages/algorithmia-enterprise/scms.md:101]: All of the images on this page use the same `alt` tag value of `Set an SCM as default`

**Comment by ryanmillerdev** [comment]
Thanks!!

**Comment by bnlb** [comment]
Very well explained! Everything else looks good. I'll approve once the alt tags are updated.

File: _pages/algorithmia-enterprise/scms.md (hunk 1 above)
Comment (ryanmillerdev) [reply _pages/algorithmia-enterprise/scms.md:101]: Good catch! Updating

Comment (ryanmillerdev) [reply _pages/algorithmia-enterprise/scms.md:101]: @bnlb Fixed!

**Review by bnlb — APPROVED** [review]
👍

**Pull request merged**